src/time-lru-set.js

```diff
@@ -0,0 +1,164 @@
+//
+// Copyright 2019 DxOS.
+//
+
+class ItemCache {
+  constructor ({ value, maxAge, onTimeout }) {
+    this._value = value;
+    this._maxAge = maxAge;
+    this._onTimeout = value => onTimeout(value);
```

Comment:
This isn't needed since it's just called directly below.,

Reply:
fixed